src/components/Shared/Filters/NullState/NullState.test.tsx

```diff
@@ -2,18 +2,48 @@ import React from 'react';
 import { ThemeProvider } from '@mui/material/styles';
 import userEvent from '@testing-library/user-event';
 import { SnackbarProvider } from 'notistack';
+import TestRouter from '__tests__/util/TestRouter';
 import TestWrapper from '__tests__/util/TestWrapper';
 import { render, waitFor } from '__tests__/util/testingLibraryReactMock';
 import { TaskModalEnum } from 'src/components/Task/Modal/TaskModal';
+import { UrlFiltersProvider } from 'src/components/common/UrlFiltersProvider/UrlFiltersProvider';
 import useTaskModal from '../../../../hooks/useTaskModal';
 import theme from '../../../../theme';
 import NullState from './NullState';
 
-const changeFilters = jest.fn();
 const openTaskModal = jest.fn();
 
 jest.mock('src/hooks/useTaskModal');
 
+interface TestComponentProps {
+  page: 'contact' | 'task';
+  totalCount: number;
+  router?: object;
+}
+
+const defaultRouter = {
+  query: {},
+  isReady: true,
+};
+
+const TestComponent: React.FC<TestComponentProps> = ({
+  page,
+  totalCount,
+  router = defaultRouter,
+}) => (
+  <TestRouter router={router}>
+    <SnackbarProvider>
+      <ThemeProvider theme={theme}>
+        <UrlFiltersProvider>
+          <TestWrapper>
```

Comment:
Let's not use `TestWrapper` when we're also using `TestRouter` because `TestWrapper` includes a router, and it is confusing to have multiple router providers.

```suggestion
          <GqlMockedProvider>
```